packages/core/src/baseclient.ts

```diff
@@ -100,6 +101,19 @@ export abstract class BaseClient<B extends Backend, O extends Options> implement
   public captureException(exception: any, hint?: EventHint, scope?: Scope): string | undefined {
     let eventId: string | undefined = hint && hint.event_id;
 
+    if (this._options.autoSessionTracking && scope) {
+      const requestSession = scope.getRequestSession();
```

Comment:
I don't see any equivalent of `getRequestSession` in the [public API of `Scope` in the Python SDK](https://github.com/getsentry/sentry-python/blob/master/sentry_sdk/scope.py).
I'm a bit uneasy with exposing this.

Also what would be special about "Request Session" over some non-request session to deserve its own getter?

We do need to keep track of the session, but we should do so without exposing new public API.

Reply:
Sure I will make these private methods then